launch/display.launch

```diff
@@ -6,7 +6,7 @@
 
   <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
 
-  <param name="publish_frequency" value="100"/>
+  <param name="publish_frequency" value="1000"/>
```

Comment:
It needs for the f/t sensor. Tf pubblisher and joint state pubblisher  should have the same sampling frequency of the f/t sensor (1KHz). Otherwise il could be some errors.